gnucobol.yaml

```
@@ -0,0 +1,46 @@
+package:
+  name: gnucobol
+  version: 3.2
+  epoch: 0
+  description: "compiler from COBOL to C"
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - gmp-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://ftp.gnu.org/pub/gnu/gnucobol/gnucobol-${{package.version}}.tar.gz
+      expected-sha256: 29f30a77176015847f0afb2e22939e39798bb4d98c7c7a26f6765930b4553c52
+
+  - uses: autoconf/configure
+    with:
+      opts: |
+        --without-db
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+update:
+  enabled: false
```

We can use release-monitor to keep a check for any future updates:
 - https://release-monitoring.org/project/8854/


```suggestion
update:
  enabled: true
  release-monitor:
    identifier: 8854
```